applications/nrf_desktop/doc/buttons_sim.rst

```diff
@@ -0,0 +1,58 @@
+| ``buttons_sim``  |                  |                |                  |                                      |
++------------------+------------------+                +------------------+--------------------------------------+
+|                  |                  |                | ``button_event`` | :ref:`fn_keys`                       |
+|                  |                  |                |                  +--------------------------------------+
+|                  |                  |                |                  | :ref:`motion` (``motion_buttons``)   |
+|                  |                  |                |                  +--------------------------------------+
+|                  |                  |                |                  | :ref:`click_detector`                |
+|                  |                  |                |                  +--------------------------------------+
+|                  |                  |                |                  | :ref:`passkey` (``passkey_buttons``) |
+|                  |                  |                |                  +--------------------------------------+
+|                  |                  |                |                  | ``buttons_sim``                      |
+|                  |                  |                |                  +--------------------------------------+
+|                  |                  |                |                  | :ref:`hid_state`                     |
++------------------+------------------+----------------+------------------+--------------------------------------+
+
+Configuration
+*************
+
+Simulated button sequence is triggered by ``button_event``.
+You have to enable and configure the :ref:`buttons` module.
+
+Set the ``CONFIG_DESKTOP_BUTTONS_SIM_ENABLE`` Kconfig option to enable the ``buttons_sim``  module.
+
+Define the output key IDs sequence in the ``buttons_sim_def.h`` file located in board-specific directory in the ``configuration`` folder.
+The mapping from the defined key ID to HID report ID and usage ID is defined in :file:`hid_keymap_def.h` (it may vary between boards).
+
+Interval between subsequent simulated button presses is defined as ``CONFIG_DESKTOP_BUTTONS_SIM_INTERVAL``.
+1 second is used by default.
+
+The simulated button sequence will be generated once by default.
+Set the ``CONFIG_DESKTOP_BUTTONS_SIM_LOOP_FOREVER`` to make sequence start from the beginning after it ends.
+
+Implementation details
+**********************
+
+The ``buttons_sim`` module generates button sequence using :c:type:`struct k_delayed_work`, that resubmits itself.
+Every call of work function submits press and release of single button from the sequence.
+
+Receiving ``button_event`` with key ID set to ``CONFIG_DESKTOP_BUTTONS_SIM_TRIGGER_KEY_ID`` stops generating the sequence (in case it is already being generated) or starts generating the sequence (otherwise).
```

Comment:
```suggestion
Receiving ``button_event`` with the key ID set to ``CONFIG_DESKTOP_BUTTONS_SIM_TRIGGER_KEY_ID`` either stops generating the sequence (in case it is already being generated) or starts generating it.
```